integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -191,33 +192,47 @@ resources:
   {{- if .Values.tieredstore.levels }}
     {{- range .Values.tieredstore.levels }}
       {{- if .mediumtype }}
+        {{- /* The mediumtype can have multiple parts like MEM,SSD */}}
         {{- if contains "," .mediumtype }}
           {{- $split := split "," .mediumtype }}
           {{- $type := .type }}
           {{- $path := .path }}
+          {{- $volumeName := .name }}
+          {{- /* A volume will be generated for each part */}}
           {{- range $key, $val := $split }}
+            {{- $mediumName := printf "%v-%v" (lower $val) (replace $key "_" "") }}
```

Comment:
nit: add comment with example of what `mediumName` resolves to

Reply:
Good advice!